Assets/Scripts/Localization/DiffMatchPatch.cs

```diff
@@ -0,0 +1,2308 @@
+﻿/*
+ * Copyright 2008 Google Inc. All Rights Reserved.
+ * Author: fraser@google.com (Neil Fraser)
+ * Author: anteru@developer.shelter13.net (Matthaeus G. Chajdas)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Diff Match and Patch
+ * http://code.google.com/p/google-diff-match-patch/
+ */
+
+ //Heavily modified and stripped by Zion Nimchuk (hcorion) for use in Project Porcupine
```

Comment:
Just so that everybody knows. I plan to strip away a lot of the functions and don't want to confuse people thinking that it is very close to the original file.